src/server/game/Entities/Vehicle/Vehicle.cpp

```diff
@@ -211,6 +211,11 @@ void Vehicle::ApplyAllImmunities()
             // why we need to apply this? we can simple add immunities to slow mechanic in DB
             _me->ApplySpellImmune(0, IMMUNITY_STATE, SPELL_AURA_MOD_DECREASE_SPEED, true);
             break;
+        case 335: // Salvaged Chopper
+        case 336: // Salvaged Siege Engine
+        case 338: // Salvaged Demolisher
+            _me->ApplySpellImmune(0, IMMUNITY_STATE, SPELL_AURA_MOD_DAMAGE_PERCENT_TAKEN, false); // Battering Ram
```

Comment:
maybe Machiavelli could tell us the reason since he added it :)

but i think it was added to prevent using http://wotlk.openwow.com/spell=20233 on vehicles. Since we cannot heal vehicles i think it's fair to disable damagereduces in general too